### DISCUSSION TITLE
Is a good idea make the maps sealed?

### DISCUSSION THREAD

**Discussion by montyclt** [nhibernate/fluent-nhibernate#422] [General] [answered]
Using `Component(e => e.Phone)`, Rider marks warning because I'm using a virtual method in a constructor and sugest me make the class sealed.

For now, I can't see any situation when someone need to extends any specified map.

Is a good idea make all maps sealed? If no, someone can explain why not, or where someone need to extends an specified map?

**Comment by hazzik** [comment, marked as answer]
There is no reasons why you cannot have the maping classes sealed.
